### PR TITLE
Ten day

### PR DESCRIPTION
**Pull request by Obleo33**

### CODE REVIEW

File: lib/components/Hourly.js [hunk 1]
```diff
@@ -9,12 +9,13 @@ const Hourly = ({ weatherHourly }) => {
...
       <div className='hourly-forcast'>
       {weatherHourly.map((obj, i) => {
-        
+
         return(
           <div className='hourly-card' key={i}>
             <p>Hour: {obj.FCTTIME.civil}</p>
             <p>Temp: {obj.temp.english + '°' }</p>
             <p>Condition: {obj.condition}</p>
+            <p>-----------------------------------------</p>
```
Comment (Obleo33) [inline lib/components/Hourly.js:18]: line breaks for visibility for now

File: lib/components/Main.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ class Main extends Component {
   render() {
     return (
       <div>
-        <Weathrly source='http://api.wunderground.com/api/395c3eaa19684a65/geolookup/hourly/forecast10day/q/' />
+        <Weathrly source='http://api.wunderground.com/api/395c3eaa19684a65/conditions/geolookup/hourly/forecast10day/q/' />
       </div>
```
Comment (Obleo33) [inline lib/components/Main.js:9]: added conditions for the current temp

File: lib/components/Hourly.js (hunk 1 above)
Comment (hmorri32) [reply lib/components/Hourly.js:18]: cool

File: lib/components/Main.js (hunk 2 above)
Comment (hmorri32) [reply lib/components/Main.js:9]: cool

**Pull request merged**